drivers/storage/portworx/component/telemetry.go

```diff
@@ -855,6 +993,476 @@ func (t *telemetry) reconcileCCMProxyConfigMap(
 	)
 }
 
+func (t *telemetry) reconcileCCMGoRolesAndRoleBindings(
```

Comment:
nit: you may put old telemetry into telemetry-old.go, the new telemetry.go can just call the APIs of the old telemetry object. in that case we dont need to rename the APIs. just something to consider, dont want to add overhead as this is already done up to you evaluate

Reply:
Yes, I like moving the old code to a different file and keeping the new components clean.